src/components/Signin.jsx

```diff
@@ -245,12 +245,12 @@ const Signin = () => {
                                     <AuthLink 
                                       color="white" bgColor="#0e76a8" 
                                       href="#" className=" btn" 
-                                      TextAlign="center" IconSize="32px"
-                                      IconColor="#0e76a8" IconFloat="left" 
-                                      IconBgColor="white" 
+                                      TextAlign="center" IconSize="18px"
+                                      IconColor="white" marginRight="5px"
+                                      IconBgColor="#0e76a8"
                                       hoverColor="white" hoverBgColor="#3168E5"
                                     >
-                                      <i className="fa fa-linkedin-square " />Signin with Linkedin
```

Comment:
Remove Space from each text. This is how your options will be aligned vertically also.

Reply:
i didnt understand , what space should i remove?